examples/vision/attention_mll_classification.py

```diff
@@ -0,0 +1,541 @@
+"""
+Title: Classification using Attention-based Deep MIL
+Author: [Mohamad Jaber](https://www.linkedin.com/in/mohamadjaber1/)
+Date created: 2021/08/16
+Last modified: 2021/08/16
+Description: A neural network model that uses attention weights on instance-level to decide the output class.
+"""
+"""
+## Introduction
+
+Instead of having individual labelled instances, this neural network model receives a single label for each bag of instances.
+The bag is labelled positive if it contains atleast one positive instance and negative if it does not contain any.
+The MIL problem is stated as the learning Bernoulli distribution of the bag label where output probability is parametermized by the algorithm.
+The attention mechanism provides insight into the contribution of each instance to the bag label.
+
+Besides the implementation of this mechanism, some **regularization** techniques, **ensemble averaging** (for stability)
+and dealing **imbalanced data** will be covered.
+
+## References
+
+- The paper [Attention-based Deep Multiple Instance Learning](https://arxiv.org/pdf/1802.04712.pdf) is authored by Maximilian Ilse,
+Jakub M. Tomczak and Max Welling.
+- Some of attention operator code implementation was inspired from https://github.com/utayao/Atten_Deep_MIL.
+- Imbalanced data [tutorial](https://www.tensorflow.org/tutorials/structured_data/imbalanced_data) by TensorFlow.
+"""
+"""
+## Setup
+"""
+
+import numpy as np
+import os
+from tqdm import tqdm
+import tempfile
+from tensorflow.python.keras.datasets import mnist, fashion_mnist
+from tensorflow.python.keras.models import Model
```

Comment:
You can access it via `keras.Model`. In general do not import individual symbols

Reply:
Updated.